manifests/base/jupyter-tensorflow-notebook-imagestream.yaml

```diff
@@ -32,6 +33,7 @@ spec:
         opendatahub.io/notebook-software: '[{"name":"CUDA","version":"11.8"},{"name":"Python","version":"v3.9"},{"name":"TensorFlow","version":"2.11"}]'
         opendatahub.io/notebook-python-dependencies: '[{"name":"TensorFlow","version":"2.11"},{"name":"Tensorboard","version":"2.11"},{"name":"Boto3","version":"1.26"},{"name":"Kafka-Python","version":"2.0"},{"name":"Kfp-tekton","version":"1.5"},{"name":"Matplotlib","version":"3.6"},{"name":"Numpy","version":"1.24"},{"name":"Pandas","version":"1.5"},{"name":"Scikit-learn","version":"1.2"},{"name":"Scipy","version":"1.10"},{"name":"Elyra","version":"3.15"}]'
         openshift.io/imported-from: quay.io/opendatahub/workbench-images
+        opendatahub.io/notebook-build-commit: "17c2e49"
       from:
         kind: DockerImage
         name: $(odh-tensorflow-gpu-notebook-image-n-1)
```

Comment:
Lets add the value to bottom version: `1.2`

`opendatahub.io/notebook-build-commit: 4c8f26e`
Image: `quay.io/opendatahub/notebooks:cuda-jupyter-tensorflow-ubi8-python-3.8-4c8f26e`